test/Constraints/closures.swift

```diff
@@ -64,19 +64,21 @@ func test13811882() {
 
 
 // <rdar://problem/21544303> QoI: "Unexpected trailing closure" should have a fixit to insert a 'do' statement
+// <https://bugs.swift.org/browse/SR-3671>
 func r21544303() {
-  var inSubcall = true
+  let val = true
+  var inSubcall = val
   {   // expected-error {{expected 'do' keyword to designate a block of statements}} {{3-3=do }}
```

Comment:
The presence of the literal `true` triggered my new `break;` and so the `{}` was parsed as part of the variable declaration, leading to `computed property must have accessors specified`.

The logic that produces `expected 'do' keyword to designate a block of statements` only looks for a trailing closure, so no longer applies with literals after my change. This didn't seem like a huge problem, but definitely an area for possible improvement.